src/index.js

```diff
@@ -305,13 +314,12 @@ export default class HyperScreen extends React.Component {
         // We are doing this to ensure that we keep the screen stale until a `reload` happens
         this.setState({ staleHeaderType });
       }
+      if (this.state.elementError) {
+        this.setState({ elementError: null });
+      }
```

Comment:
shouldn't the error stated cleared before the request is emitted?

Reply:
If we clear the error state before the request was made the screen will re-render and the banner would flicker in the case where the error still persists. This might be a good user experience imo, what do you think?

https://user-images.githubusercontent.com/28518512/217935762-2aac5024-557e-4203-a7bc-5f3516c7a292.mp4

Reply:
Got it, I think you're right. I was looking at this in an optimistically update point of view, assuming that the next call would succeed but that may not be true.